### ISSUE TITLE
Unrestrict notification actions

### ISSUE THREAD

**Issue by Madis0** [TeamNewPipe/NewPipe#4399]
### Checklist

- [x] I checked, but didn't find any duplicates of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want
With the new notification system, the buttons are customizable. However, I don't understand why the selections are restricted per button so I suggest removing this limitation.

#### Is your feature request related to a problem? Please describe it
The first button can only be "previous" or "rewind". The second button can only be "rewind" or "play/pause", etc.

#### How will you/everyone benefit from this feature?
Everyone could fully customize the notification by using any action in any slot.

**Comment by opusforlife2** [comment]
@Stypox Yeah, I don't know why cool-student did it this way, either. It would be trivial to add all options to all butttons, right?

**Comment by vkay94** [comment]
I wouldn't say that it's trivial: it is usual for a media notification that the "left" is associated with "rewird/previous/backwards" and "right" accordingly to "next/forwards".

**Comment by Madis0** [comment]
Unless you do a different setup, that is.
E.g. ⏮️⏪⏯️⏩⏭️, ⏮️⏭️⏪⏩⏯️, ⏯️⏪⏩🔁🔀 and so on.

**Comment by opusforlife2** [comment]
@vkay94 Trivial from a coding perspective. Whether or not it is ergonomic would depend from user to user.

**Comment by vkay94** [comment]
@Madis0 Hmm, I see that the second variant is possible which is not so good. The first and third are okay imo. I just think that mixing the directions breaks the usual concept.

@opusforlife2 Yeah that should be done easily even if I haven't looked into it :')
